docs/upgrade.md

```diff
@@ -55,6 +55,10 @@ Bubbled child events no longer pass the `childView` implicitly and only pass the
 arguments passed as part of `triggerMethod`. This means that the arguments
 passed to `onEvent` and `onChildviewEvent` are now identical.
 
+In Marionette 2, `childEvents` where bound on every event. In Marionette 3,
+`childViewEvents` are bound once and cached. This means that you cannot add new
```

Comment:
That's a good spot. My understanding is that you can do it manually so perhaps this should change to "Marionette will not automatically listen to any new new events after the view has been created". What are your thoughts?